src/components/QRShare/index.tsx

```diff
@@ -39,16 +55,18 @@ function QRShare({url, title, subtitle, logo, svgLogo, svgLogoFillColor, logoBac
 
     return (
         <View
-            style={styles.shareCodeContainer}
+            style={[styles.shareCodeContainer, additionalStyles, isFromDownloadPage ? styles.shareCodeContainerDownloadPadding : styles.shareCodeContainerPadding]}
```

Comment:
`additionalStyles` should be last in the array so we overwrite previous defined styles